templates/repo/issue/view_content/comments_delete_time.tmpl

```diff
@@ -1,18 +1,20 @@
-{{if .comment.Time}} {{/* compatibility with time comments made before v1.14 */}}
-	{{if (not .comment.Time.Deleted)}}
-		{{if (or .ctxData.IsAdmin (and .ctxData.IsSigned (eq .ctxData.SignedUserID .comment.PosterID)))}}
-			<span class="gt-float-right">
-				<div class="ui mini modal issue-delete-time-modal" data-id="{{.comment.Time.ID}}">
-					<form method="post" class="delete-time-form" action="{{.ctxData.RepoLink}}/issues/{{.ctxData.Issue.Index}}/times/{{.comment.TimeID}}/delete">
-						{{.ctxData.CsrfTokenHtml}}
-					</form>
-					<div class="header">{{ctx.Locale.Tr "repo.issues.del_time"}}</div>
-					{{template "base/modal_actions_confirm"}}
-				</div>
-				<button class="ui icon button compact mini issue-delete-time" data-id="{{.comment.Time.ID}}" data-tooltip-content="{{ctx.Locale.Tr "repo.issues.del_time"}}">
-					{{svg "octicon-trash"}}
-				</button>
-			</span>
+{{if .ctxData.Repository.IsTimetrackerEnabled .ctxData.Context}}
```

Comment:
Instead of increasing the indentation, what about using `and` instead?

Reply:
+1, I would suggest do not indent.